src/components/ReceiverDerivedAccount.tsx

```diff
@@ -54,21 +50,30 @@ const ReceiverDerivedAccount = () => {
   if (derivedReceiverAccount) {
     const shortUnformattedReceiver = shortenItem(unformattedReceiverAddress || '');
     return (
-      <div className={classes.derived}>
+      <div className={classes.accountCompanion}>
         <Account
-          accountName={shortUnformattedReceiver}
+          friendlyName={shortUnformattedReceiver}
           value={receiverAddress!}
           chain={targetChain}
           isDerived
           hideAddress
+          withTooltip
         />
       </div>
     );
   }
 
+  if (!receiverAddress) {
+    return (
+      <div className={classes.accountCompanion}>
+        <AccountDisplay />
+      </div>
+    );
+  }
+
   return (
-    <div className={classes.derived}>
-      <AccountDisplay addressKind={AddressKind.COMPANION} derived />
+    <div className={classes.accountCompanion}>
+      <AccountDisplay address={receiverAddress!} addressKind={AddressKind.NATIVE} isDerived withTooltip />
```

Comment:
why are you changing the `addressKind` ? shouldn't be 
`<AccountDisplay address={receiverAddress!} addressKind={AddressKind.COMPANION} isDerived withTooltip />` ?

Reply:
I understood "companion" is an alias for "derived". In this case we send directly to address user has inputed, not to derived from this input, so it's not "companion". Am I confused?

Reply:
exactly, but you are replacing NATIVE by COMPANION and i think it should remain as COMPANION right?

Reply:
This is changes for affected component in affected state:

_before_
<img width="479" alt="Screenshot 2021-05-14 at 16 45 02" src="https://user-images.githubusercontent.com/8782596/118287626-ea7b9980-b4d3-11eb-89d7-63087bd84856.png">

_after_
<img width="478" alt="Screenshot 2021-05-14 at 16 44 22" src="https://user-images.githubusercontent.com/8782596/118287622-e94a6c80-b4d3-11eb-893b-3bffff7fd7b0.png">

Reply:
I've just removed isDerived prop

Reply:
As the UX was enhanced . This change is valid.